sections/sprint-4/updated-uppaal.tex

```diff
@@ -164,32 +164,39 @@ \subsection*{UDP host templates}
+
+\subsubsection{Verification of the application-level model}
+The property held true for the model an the trace that was produced can be seen on \autoref{fig:sprint4-verification-application-level}.
+As can be seen on the trace, all of the clients go through the process of connecting without timing out.
+When all of the clients have connected, the \uppProc{Host} starts the game through the \uppSync{TCP_start_game} synchronization and the host and all of the clients transition to the \uppLoc{InGame} location.
 \begin{figure}[H]
     \centering
-    \includegraphics[width=1\linewidth]{sprint4/connected-trace.png}
-    \caption{Example of a trace where both \uppProc{Client} processes connect without any problems.}
-    \label{fig:sprint4-happy-trace}
+    \includegraphics[width=0.8\linewidth]{sprint4/application-level-verification.png}
+    \caption{Verification trace of the application-level model.}
+    \label{fig:sprint4-verification-application-level}
 \end{figure}
 
-\subsection*{Trace with a timeout}
-A trace where it times out once while connecting can be seen on \autoref{fig:sprint4-timeout-trace}.
-\uppTemp{Client(1)} connects without any issues, but \uppTemp{Client(0)} times out once while connecting.
-This can be seen on the trace of \uppProc{TCP_client(0)}. When it sends the \uppOut{tcp_sync[0]} it times out because it does not get a response from the \uppProc{TCP_host(1)}.
-It then resends the message successfully and continues with connecting.
+\subsubsection{Verification of the Full model}
+The property also held true for the full model and the trace can be seen on \autoref{fig:sprint4-verification-full-model}.
+As can be seen in the trace all of the clients connect through the three-way handshake.
+All of the clients successfully transition to \uppLoc{Lobby} without any timeouts.
+When all of the clients have connected the host makes all of the clients transition to \uppLoc{InGame} through TCP.
+After all of the clients have transitioned to the \uppLoc{InGame} location the host also transitions to \uppLoc{InGame} and the game starts.
 \begin{figure}[H]
     \centering
-    \includegraphics[width=1\linewidth]{sprint4/timeout-trace.png}
-    \caption{Example of a trace where a \uppProc{TCP_host} times out while connecting.}
-    \label{fig:sprint4-timeout-trace}
+    \includegraphics[width=0.8\linewidth]{sprint4/full-model-verification.png}
+    \caption{Verification trace of the full model.}
+    \label{fig:sprint4-verification-full-model}
 \end{figure}
+
+\subsection{Conclusion}
+The models introduced in this section were mostly used to create an overview of the network protocol and the flow of the game.
+All the verification done on the models was done to verify that the models were behaving as expected and not to test the actual system.
+It might have been useful to use to models to verify the actual implementation of the network protocol and the flow of the application, but this was deemed to not be a priority for the project.
```

Comment:
Sammenlign trace fra de to modeller og fortæl hvad forskellen er, hvad kan den ene udtrykke som den anden ikke kan?